lib/kazoo/consumergroup.rb

```diff
@@ -237,23 +338,21 @@ def registered?
         stat.fetch(:stat).exists?
       end
 
-      def register(subscription)
+      def register(subscription_deprecated = nil)
+        # Don't provide the subscription here, but provide it when instantiating the consumer instance.
```

Comment:
This is here just for backwards compatibility. Once I have updated kafka-consumer, it can be removed.